build_mesos

```diff
@@ -214,6 +209,12 @@ function os_release {
       err "/etc/redhat-release not like: <distro> release <version> (<remark>)"
     fi
   fi
+  msg "Trying /etc/os-release..."
```

Comment:
Just a quick question, any reason to move this from the top to here? (Reason I'm asking is I'd rather avoid unnecessary changes so if someone looks through the history they don't get confused 😄)

Reply:
:-) Yes! On almalinux there is also a os-release file. So, if it was found, it exit the os_release release function without to check the redhat-release file.